src/utils/psbt.js

```diff
@@ -112,11 +133,21 @@ async function broadcastPsbt(psbt) {
 }
 
 export async function signAndBroadcastUtxo({ pubKey, utxo, destinationBtcAddress, sendFeeRate }) {
-    const inputAddressInfo = await getAddressInfo(pubKey);
+    const inputAddressInfo = getAddressInfo(pubKey);
     const { psbt, inputParams } = createPsbt({ utxo, inputAddressInfo, destinationBtcAddress, sendFeeRate });
     const signed = await getSignedPsbt({ psbt, inputParams, inputAddressInfo, utxo });
     // Finalize the PSBT. Note that the transaction will not be broadcast to the Bitcoin network yet.
     signed.finalizeAllInputs();
     // Send it!
     return broadcastPsbt(signed);
 }
+
+export async function createAndSignPsbtForBoost({ pubKey, utxo, destinationBtcAddress }) {
+    const inputAddressInfo = getAddressInfo(pubKey);
+    const { psbt, inputParams } = createPsbtForBoost({ utxo, inputAddressInfo, destinationBtcAddress });
```

Comment:
nit: can merge createPsbtForBoost inside of createAndSignPsbtForBoost()

Think createPsbtForBoost() might be over-abstracting